engine/engine/content/builtins/manifests/ios/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>NSPrivacyTracking</key>
+        <false/>
+        <key>NSPrivacyTrackingDomains</key>
+        <array>
+        </array>
+        <key>NSPrivacyCollectedDataTypes</key>
+        <array>
+        </array>
+        <key>NSPrivacyAccessedAPITypes</key>
+        <array>
+                <!-- fstat() in mount_mmap.cpp -->
+                <dict>
+                        <key>NSPrivacyAccessedAPIType</key>
+                        <string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+                        <key>NSPrivacyAccessedAPITypeReasons</key>
+                        <array>
+                                <string>C617.1</string>
+                        </array>
+                </dict>
+                <!-- NSPrivacyAccessedAPICategorySystemBootTime() in Remotery.c -->
+                <dict>
```

Comment:
Does this API end up in release build? I thought `Remotery.c` is debug only

Reply:
"But I suppose you could make a release with Remotery available right"

But then I would also assume that the developer would add a custom privacy manifest?
But if it doesn't matter, then sure.

Reply:
Yes, the developer can copy the file and modify it. Or we should include one file for release and one for debug builds? Or remove this value from the default?

I honestly don't think it matters much and we can change it later.

Reply:
It would be nice if we be able to use info from app manifest to keep or remove blocks from there. Something similar we do based on info from game.project using mustache tamplates.


something like this in PrivacyInfo.xcprivacy
```
...
{{#appmanifest_libs.profilerext_null}}
<dict>
         <key>NSPrivacyAccessedAPIType</key>
         <string>NSPrivacyAccessedAPICategorySystemBootTime</string>
         <key>NSPrivacyAccessedAPITypeReasons</key>
         <array>
                 <string>35F9.1</string>
         </array>
 </dict>
{{/appmanifest_libs.profilerext_null}}
...
```

But maybe it's something we should think about in a separate iteration